app.rb

```diff
@@ -0,0 +1,99 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+
+# rubocop:disable Style/GlobalVars
+$person_list = []
+$rentals_list = []
+$students_list = []
+$teachers_list = []
+$books_list = []
+
+def list_all_books
+  return unless defined?($books_list)
+
+  $books_list.each do |book|
+    puts "[Book] Title: #{book.title} Author: #{book.author}"
+  end
+end
+
+def list_all_people
+  return unless defined?($person_list)
+
+  $person_list.each do |person|
+    if defined?(person.specialization)
+      puts "[Teacher] Name: #{person.name}, ID: #{person.id} Age: #{person.age}"
+    else
+      puts "[Student] Name: #{person.name}, ID: #{person.id} Age: #{person.age}"
+    end
+  end
+end
+
+def create_student(age, name, _specialization, classroom)
+  student = Student.new(age, name, parent_permission, classroom)
```

Comment:
- [ ] I am able to see your UI menu but when I am trying to do the different actions Creating Book, Creating Person, etc ... I am getting errors. Please make sure to fix all the errors in order to make your app works as expected :+1: 

![Screenshot_2023-05-20_18-26-11](https://github.com/Mutai-Gilbert/School-Library/assets/6615394/28d89124-ad0b-46a8-9b43-150395362cc6)